### PR TITLE
ticdc: add row filter config

### PR DESCRIPTION
**Pull request by asddongmen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -630,6 +630,24 @@ ignore-txn-start-ts = [1, 2]
...
+[[filter.event-filters]]
+matcher = ["test.fruit"] # 该事件过滤器只会应用于 test.fruit 这张表
+ignore-event = ["drop table"] # 忽略 drop table 事件
+ignore-sql = ["delete"] # 忽略 delete DML
+ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # 忽略列 price 值大于 1000, 并且 origin 列值为 'no where' 的 insert DML
+
+# 注意: TiCDC v6.1.0 版本之后已经移除该配置项，用户配置该项时不会生效。
 [mounter]
 # mounter 线程数，用于解码 TiKV 输出的数据
 worker-num = 16
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:653]: 如果是已经移除的配置项，把这几行都注释掉吧

File: ticdc/manage-ticdc.md [hunk 2]
```diff
@@ -650,10 +668,68 @@ dispatchers = [
 protocol = "canal-json"
 ```
 
+### Event Filter 配置规则
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:671]:
```suggestion
### Event Filter 配置规则 <span class="version-mark">从 v6.2.0 版本开始引入</span>
```

File: ticdc/manage-ticdc.md [hunk 3]
```diff
@@ -650,10 +668,68 @@ dispatchers = [
...
+
+TiCDC 在 v6.2.0 中新增了事件过滤器功能，用户可以通过配置该规则来过滤指定 DML 和 DDL 事件。
+
+一个事件过滤器的配置规则的例子如下：
+
+```toml
+[[filter.event-filters]]
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:678]:
```suggestion
[filter]
# 事件过滤器的规则应该写在 filter 配置项之下

[[filter.event-filters]]
```

File: ticdc/manage-ticdc.md [hunk 4]
```diff
@@ -650,10 +668,68 @@ dispatchers = [
...
+matcher = ["test.worker"] # 该过滤规则只会对 test 库中的 worker 表进行应用
+ignore-event = ["insert"] # 过滤掉 insert 事件
+ignore-sql = ["^drop", "add column"] # 过滤掉以 "drop" 开头或者 包含 "add column" 的 DDL
+ignore-delete-value-expr = "name = 'john'" # 过滤掉包含列名为 name 且值为 'john' 的 delete DML
+ignore-insert-value-expr = "id >= 100" # 过滤掉包含列名为 id 且值大于等于 100 的 insert DML 
+ignore-update-old-value-expr = "age < 18" # 过滤掉包含列名为 age 且旧值小于 18 的 update DML
+ignore-update-new-value-expr = "gender = 'male'" # 过滤掉包含列名为 gender 且新值等于 male 的 update DML
+```
+
+事件过滤器的规则应该写在 filter 配置项之下，具体写法可以参考上面的同步任务配置文件描述。
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:688]:
```suggestion
事件过滤器的规则应该写在 filter 配置项之下，具体写法可以参考[同步任务配置文件描述](#同步任务配置文件描述)。
```

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -630,6 +630,24 @@ ignore-txn-start-ts = [1, 2]
 # 过滤规则语法：https://docs.pingcap.com/zh/tidb/stable/table-filter#表库过滤语法
 rules = ['*.*', '!test.*']
 
+# 事件过滤器规则 (TiCDC v6.2.0 新增功能)
+# 事件过滤器的详细配置规则在下方的 Event Filter 配置规则中描述
+[[filter.event-filters]]
+matcher = ["test.worker"] # matcher 是一个白名单，表示该过滤规则只会对 test 库中的 worker 表进行应用
+ignore-event = ["insert"] # 过滤掉 insert 事件
+ignore-sql = ["^drop", "add column"] # 过滤掉以 "drop" 开头或者 包含 "add column" 的 DDL
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:638]:
```suggestion
ignore-sql = ["^drop", "add column"] # 过滤掉以 "drop" 开头或者包含 "add column" 的 DDL
```

File: ticdc/manage-ticdc.md [hunk 6]
```diff
@@ -630,6 +630,24 @@ ignore-txn-start-ts = [1, 2]
...
+ignore-delete-value-expr = "name = 'john'" # 过滤掉包含列名为 name 且值为 'john' 的 delete DML
+ignore-insert-value-expr = "id >= 100" # 过滤掉包含列名为 id 且值大于等于 100 的 insert DML 
+ignore-update-old-value-expr = "age < 18" # 过滤掉包含列名为 age 且旧值小于 18 的 update DML
+ignore-update-new-value-expr = "gender = 'male'" # 过滤掉包含列名为 gender 且新值等于 male 的 update DML
+
+[[filter.event-filters]]
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:644]: L635 和 L644 是重复的吗？

File: ticdc/manage-ticdc.md [hunk 7]
```diff
@@ -650,10 +668,68 @@ dispatchers = [
...
+
+TiCDC 在 v6.2.0 中新增了事件过滤器功能，用户可以通过配置该规则来过滤指定 DML 和 DDL 事件。
+
+一个事件过滤器的配置规则的例子如下：
+
+```toml
+[[filter.event-filters]]
+matcher = ["test.worker"] # 该过滤规则只会对 test 库中的 worker 表进行应用
+ignore-event = ["insert"] # 过滤掉 insert 事件
+ignore-sql = ["^drop", "add column"] # 过滤掉以 "drop" 开头或者 包含 "add column" 的 DDL
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:681]:
```suggestion
ignore-sql = ["^drop", "add column"] # 过滤掉以 "drop" 开头或者包含 "add column" 的 DDL
```

File: ticdc/manage-ticdc.md [hunk 8]
```diff
@@ -650,10 +668,68 @@ dispatchers = [
...
+ignore-delete-value-expr = "name = 'john'" # 过滤掉包含列名为 name 且值为 'john' 的 delete DML
+ignore-insert-value-expr = "id >= 100" # 过滤掉包含列名为 id 且值大于等于 100 的 insert DML 
+ignore-update-old-value-expr = "age < 18" # 过滤掉包含列名为 age 且旧值小于 18 的 update DML
+ignore-update-new-value-expr = "gender = 'male'" # 过滤掉包含列名为 gender 且新值等于 male 的 update DML
+```
+
+事件过滤器的规则应该写在 filter 配置项之下，具体写法可以参考上面的同步任务配置文件描述。
+
+#### 配置参数说明
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:690]:
```suggestion
配置参数说明：
```
这里只有一个四级标题，一般不建议有孤立的标题。

File: ticdc/manage-ticdc.md [hunk 9]
```diff
@@ -650,10 +668,68 @@ dispatchers = [
...
+| rename table    | DDL  | |匹配 rename table event    |
+| truncate table  | DDL  | |匹配 truncate table event  |
+| alter table     | DDL  | |匹配 alter table event (包含 alter table 的所有子句和 create/drop index)     |
+| add table partition    | DDL  | |匹配 add table partition event     |
+| drop table partition    | DDL  | |匹配 drop table partition event     |
+| truncate table partition    | DDL  | |匹配 truncate table partition event     |
+| create view     | DDL  | |匹配 create view event     |
+| drop view     | DDL  | |匹配 drop view event     |
+
+- `ignore-sql`: 要过滤的 DDL 语句的正则表达式，它是一个字符串数组，可以配置多条正则表达式。(该配置仅对 DDL 事件生效)
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:715]:
```suggestion
- `ignore-sql`: 要过滤的 DDL 语句的正则表达式，它是一个字符串数组，可以配置多条正则表达式。该配置仅对 DDL 事件生效。
```

File: ticdc/manage-ticdc.md [hunk 10]
```diff
@@ -650,10 +668,68 @@ dispatchers = [
...
+- `ignore-insert-value-expr`: 配置一个表达式，对 INSERT 类型的 DML 事件带有的值生效。
+- `ignore-update-old-value-expr`: 配置一个表达式，对 UPDATE 类型的 DML 事件更新的旧值生效。
+- `ignore-update-new-value-expr`: 配置一个表达式，对 UPDATE 类型的 DML 事件更新的新值生效。
+
+> **注意：**
+>
+> 1. TiDB 在更新聚簇索引的列值时会将一个 UPDATE 事件拆分成为 DELETE 和 INSERT 事件，TiCDC 无法正确的将该类事件正确的识别为 UPDATE 事件，因此无法正确的进行过滤。
+>
+> 2. 在配置 SQL 表达式时请确保符合 matcher 规则的所有表中均包含了对应表达式中的所有列，否则同步任务将无法创建成功。此外，若在同步的过程中表的结构发生变化，不再包含 SQL 表达式中的列，那么同步任务将会进入无法自动恢复的错误状态，需要用户手动修改配置并进行恢复操作。
+>
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:726]:
```suggestion
> - TiDB 在更新聚簇索引的列值时，会将一个 UPDATE 事件拆分成为 DELETE 和 INSERT 事件，TiCDC 无法正确的将该类事件正确地识别为 UPDATE 事件，因此无法正确地进行过滤。
>
> - 在配置 SQL 表达式时，请确保符合 matcher 规则的所有表中均包含了对应表达式中的所有列，否则同步任务将无法创建成功。此外，若在同步的过程中表的结构发生变化，不再包含 SQL 表达式中的列，那么同步任务将会进入无法自动恢复的错误状态，需要用户手动修改配置并进行恢复操作。
```

File: ticdc/manage-ticdc.md (hunk 6 above)
Comment (asddongmen) [reply ticdc/manage-ticdc.md:644]: 不是，这是表格数组的写法，event-filters 是一个表格，它是 filter 的一个数组元素，一个 filter 项可以配置多个 event-filters 。

File: ticdc/manage-ticdc.md (hunk 1 above)
Comment (asddongmen) [reply ticdc/manage-ticdc.md]: done

**Review by ran-huang — APPROVED** [review]
LGTM

File: ticdc/manage-ticdc.md [hunk 11]
```diff
@@ -630,9 +630,24 @@ ignore-txn-start-ts = [1, 2]
 # 过滤规则语法：https://docs.pingcap.com/zh/tidb/stable/table-filter#表库过滤语法
 rules = ['*.*', '!test.*']
 
-[mounter]
-# mounter 线程数，用于解码 TiKV 输出的数据
-worker-num = 16
+# 事件过滤器规则 (TiCDC v6.2.0 新增功能)
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:633]:
```suggestion
# 事件过滤器规则
```

File: ticdc/manage-ticdc.md [hunk 12]
```diff
@@ -630,9 +630,24 @@ ignore-txn-start-ts = [1, 2]
...
+ignore-event = ["insert"] # 过滤掉 insert 事件
+ignore-sql = ["^drop", "add column"] # 过滤掉以 "drop" 开头或者包含 "add column" 的 DDL
+ignore-delete-value-expr = "name = 'john'" # 过滤掉包含列名为 name 且值为 'john' 的 delete DML
+ignore-insert-value-expr = "id >= 100" # 过滤掉包含列名为 id 且值大于等于 100 的 insert DML 
+ignore-update-old-value-expr = "age < 18" # 过滤掉包含列名为 age 且旧值小于 18 的 update DML
+ignore-update-new-value-expr = "gender = 'male'" # 过滤掉包含列名为 gender 且新值等于 male 的 update DML
+
+# 第二个事件过滤器规则
+[[filter.event-filters]]
+matcher = ["test.fruit"] # 该事件过滤器只会应用于 test.fruit 这张表
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:647]:
```suggestion
matcher = ["test.fruit"] # 该事件过滤器只应用于 test.fruit 表
```

File: ticdc/manage-ticdc.md [hunk 13]
```diff
@@ -650,10 +665,70 @@ dispatchers = [
 protocol = "canal-json"
 ```
 
+### Event Filter 配置规则 <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
+TiCDC 在 v6.2.0 中新增了事件过滤器功能，用户可以通过配置该规则来过滤符合指定条件的 DML 和 DDL 事件。
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:670]:
```suggestion
TiCDC 在 v6.2.0 中新增了事件过滤器功能，你可以通过配置该规则来过滤符合指定条件的 DML 和 DDL 事件。
```

File: ticdc/manage-ticdc.md [hunk 14]
```diff
@@ -650,10 +665,70 @@ dispatchers = [
...
+
+一个事件过滤器的配置规则的例子如下：
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:672]:
```suggestion
以下是事件过滤器的配置规则示例：
```

File: ticdc/manage-ticdc.md [hunk 15]
```diff
@@ -650,10 +665,70 @@ dispatchers = [
...
+
+```toml
+[filter]
+# 事件过滤器的规则应该写在 filter 配置项之下，可以同时配置多个事件过滤器。
+
+[[filter.event-filters]]
+matcher = ["test.worker"] # 该过滤规则只会对 test 库中的 worker 表进行应用
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:679]:
```suggestion
matcher = ["test.worker"] # 该过滤规则只应用于 test 库中的 worker 表
```

File: ticdc/manage-ticdc.md [hunk 16]
```diff
@@ -650,10 +665,70 @@ dispatchers = [
...
+
+- `ignore-sql`: 要过滤的 DDL 语句的正则表达式，它是一个字符串数组，可以配置多条正则表达式。该配置仅对 DDL 事件生效。
+- `ignore-delete-value-expr`: 配置一个表达式，对 DELETE 类型的 DML 事件带有的值生效。
+- `ignore-insert-value-expr`: 配置一个表达式，对 INSERT 类型的 DML 事件带有的值生效。
+- `ignore-update-old-value-expr`: 配置一个表达式，对 UPDATE 类型的 DML 事件更新的旧值生效。
+- `ignore-update-new-value-expr`: 配置一个表达式，对 UPDATE 类型的 DML 事件更新的新值生效。
+
+> **注意：**
+>
+> - TiDB 在更新聚簇索引的列值时，会将一个 UPDATE 事件拆分成为 DELETE 和 INSERT 事件，TiCDC 无法正确的将该类事件正确地识别为 UPDATE 事件，因此无法正确地进行过滤。
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:723]:
```suggestion
> - TiDB 在更新聚簇索引的列值时，会将一个 UPDATE 事件拆分成为 DELETE 和 INSERT 事件，TiCDC 无法将该类事件识别为 UPDATE 事件，因此无法正确地进行过滤。
```

File: ticdc/manage-ticdc.md [hunk 17]
```diff
@@ -650,10 +665,70 @@ dispatchers = [
...
+>
+> - 在配置 SQL 表达式时，请确保符合 matcher 规则的所有表中均包含了对应表达式中的所有列，否则同步任务将无法创建成功。此外，若在同步的过程中表的结构发生变化，不再包含 SQL 表达式中的列，那么同步任务将会进入无法自动恢复的错误状态，需要用户手动修改配置并进行恢复操作。
```
Comment (hfxsd) [inline ticdc/manage-ticdc.md:725]:
```suggestion
> - 在配置 SQL 表达式时，请确保符合 matcher 规则的所有表中均包含了对应表达式中的所有列，否则同步任务将无法创建成功。此外，若在同步的过程中表的结构发生变化，不再包含 SQL 表达式中的列，那么同步任务将会进入无法自动恢复的错误状态，你需要手动修改配置并进行恢复操作。
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: ticdc/manage-ticdc.md [hunk 18]
```diff
@@ -650,10 +665,70 @@ dispatchers = [
...
+| rename table    | DDL  | |匹配 rename table event    |
+| truncate table  | DDL  | |匹配 truncate table event  |
+| alter table     | DDL  | |匹配 alter table event (包含 alter table 的所有子句和 create/drop index)     |
+| add table partition    | DDL  | |匹配 add table partition event     |
+| drop table partition    | DDL  | |匹配 drop table partition event     |
+| truncate table partition    | DDL  | |匹配 truncate table partition event     |
+| create view     | DDL  | |匹配 create view event     |
+| drop view     | DDL  | |匹配 drop view event     |
+
+- `ignore-sql`: 要过滤的 DDL 语句的正则表达式，它是一个字符串数组，可以配置多条正则表达式。该配置仅对 DDL 事件生效。
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:715]:
```suggestion
- `ignore-sql`：要过滤的 DDL 语句的正则表达式。该参数接受一个字符串数组，数组中可以配置多条正则表达式。该配置仅对 DDL 事件生效。
```

File: ticdc/manage-ticdc.md [hunk 19]
```diff
@@ -650,10 +665,70 @@ dispatchers = [
...
+- `ignore-delete-value-expr`: 配置一个表达式，对 DELETE 类型的 DML 事件带有的值生效。
+- `ignore-insert-value-expr`: 配置一个表达式，对 INSERT 类型的 DML 事件带有的值生效。
+- `ignore-update-old-value-expr`: 配置一个表达式，对 UPDATE 类型的 DML 事件更新的旧值生效。
+- `ignore-update-new-value-expr`: 配置一个表达式，对 UPDATE 类型的 DML 事件更新的新值生效。
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:719]: 这样是不是更贴切一些？参数是对 event 生效，不是对这些值生效。
```suggestion
- `ignore-delete-value-expr`: 配置一个表达式，对带有指定值的 DELETE 类型的 DML 事件生效。
- `ignore-insert-value-expr`: 配置一个表达式，对带有指定值的 INSERT 类型的 DML 事件生效。
- `ignore-update-old-value-expr`: 配置一个表达式，对带有指定旧值的 UPDATE 类型的 DML 事件生效。
- `ignore-update-new-value-expr`: 配置一个表达式，对带有指定新值的 UPDATE 类型的 DML 事件生效。
```

File: ticdc/manage-ticdc.md [hunk 20]
```diff
@@ -650,10 +665,70 @@ dispatchers = [
...
+
+> **注意：**
+>
+> - TiDB 在更新聚簇索引的列值时，会将一个 UPDATE 事件拆分成为 DELETE 和 INSERT 事件，TiCDC 无法将该类事件识别为 UPDATE 事件，因此无法正确地进行过滤。
+>
+> - 在配置 SQL 表达式时，请确保符合 matcher 规则的所有表中均包含了对应表达式中的所有列，否则同步任务将无法创建成功。此外，若在同步的过程中表的结构发生变化，不再包含 SQL 表达式中的列，那么同步任务将会进入无法自动恢复的错误状态，你需要手动修改配置并进行恢复操作。
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:725]: “SQL 表达式”是指 `ignore-sql` 还是其他的表达式？

File: ticdc/manage-ticdc.md [hunk 21]
```diff
@@ -630,9 +630,24 @@ ignore-txn-start-ts = [1, 2]
...
+# 事件过滤器的详细配置规则在下方的 Event Filter 配置规则中描述
+# 第一个事件过滤器规则
+[[filter.event-filters]]
+matcher = ["test.worker"] # matcher 是一个白名单，表示该过滤规则只应用于 test 库中的 worker 表
+ignore-event = ["insert"] # 过滤掉 insert 事件
+ignore-sql = ["^drop", "add column"] # 过滤掉以 "drop" 开头或者包含 "add column" 的 DDL
+ignore-delete-value-expr = "name = 'john'" # 过滤掉包含列名为 name 且值为 'john' 的 delete DML
+ignore-insert-value-expr = "id >= 100" # 过滤掉包含列名为 id 且值大于等于 100 的 insert DML 
+ignore-update-old-value-expr = "age < 18" # 过滤掉包含列名为 age 且旧值小于 18 的 update DML
+ignore-update-new-value-expr = "gender = 'male'" # 过滤掉包含列名为 gender 且新值等于 male 的 update DML
```
[the remark this answers is not in the document]
Comment (asddongmen) [reply ticdc/manage-ticdc.md:643]: 嗯，谢谢~

File: ticdc/manage-ticdc.md (hunk 20 above)
Comment (asddongmen) [reply ticdc/manage-ticdc.md:725]: 指定是 `ignore-insert-value-expr` 这些配置项的表达式。

Comment (ran-huang) [reply ticdc/manage-ticdc.md:725]: 可以在文档再明确一点吗？

File: ticdc/manage-ticdc.md [hunk 22]
```diff
@@ -650,10 +665,70 @@ dispatchers = [
...
+# 事件过滤器的规则应该写在 filter 配置项之下，可以同时配置多个事件过滤器。
+
+[[filter.event-filters]]
+matcher = ["test.worker"] # 该过滤规则只应用于 test 库中的 worker 表
+ignore-event = ["insert"] # 过滤掉 insert 事件
+ignore-sql = ["^drop", "add column"] # 过滤掉以 "drop" 开头或者包含 "add column" 的 DDL
+ignore-delete-value-expr = "name = 'john'" # 过滤掉包含列名为 name 且值为 'john' 的 delete DML
+ignore-insert-value-expr = "id >= 100" # 过滤掉包含列名为 id 且值大于等于 100 的 insert DML 
+ignore-update-old-value-expr = "age < 18" # 过滤掉包含列名为 age 且旧值小于 18 的 update DML
+ignore-update-new-value-expr = "gender = 'male'" # 过滤掉包含列名为 gender 且新值等于 male 的 update DML
```
Comment (ran-huang) [inline ticdc/manage-ticdc.md:685]:
```suggestion
ignore-delete-value-expr = "name = 'john'" # 过滤掉包含 name = 'john' 条件的 delete DML
ignore-insert-value-expr = "id >= 100" # 过滤掉包含 id >= 100 条件的 insert DML 
ignore-update-old-value-expr = "age < 18" # 过滤掉旧值 age < 18 的 update DML
ignore-update-new-value-expr = "gender = 'male'" # 过滤掉新值 gender = 'male' 的 update DML
```

File: ticdc/manage-ticdc.md (hunk 20 above)
Comment (asddongmen) [reply ticdc/manage-ticdc.md:725]: 明早改

Comment (asddongmen) [reply ticdc/manage-ticdc.md]: done PTAL

**Comment by ran-huang** [comment]
@leoppro PTAL. Thanks!

**Review by nongfushanquan — APPROVED** [review]

File: ticdc/manage-ticdc.md [hunk 23]
```diff
@@ -650,10 +665,70 @@ dispatchers = [
...
+# 事件过滤器的规则应该写在 filter 配置项之下，可以同时配置多个事件过滤器。
+
+[[filter.event-filters]]
+matcher = ["test.worker"] # 该过滤规则只应用于 test 库中的 worker 表
+ignore-event = ["insert"] # 过滤掉 insert 事件
+ignore-sql = ["^drop", "add column"] # 过滤掉以 "drop" 开头或者包含 "add column" 的 DDL
+ignore-delete-value-expr = "name = 'john'" # 过滤掉包含 name = 'john' 条件的 delete DML
+ignore-insert-value-expr = "id >= 100" # 过滤掉包含 id >= 100 条件的 insert DML 
+ignore-update-old-value-expr = "age < 18" # 过滤掉旧值 age < 18 的 update DML
+ignore-update-new-value-expr = "gender = 'male'" # 过滤掉新值 gender = 'male' 的 update DML
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:685]:
```suggestion
ignore-update-new-value-expr = "gender = 'male' and age > 18" # 过滤掉新值 gender = 'male' 的 update DML
```

File: ticdc/manage-ticdc.md [hunk 24]
```diff
@@ -650,10 +665,70 @@ dispatchers = [
...
+[filter]
+# 事件过滤器的规则应该写在 filter 配置项之下，可以同时配置多个事件过滤器。
+
+[[filter.event-filters]]
+matcher = ["test.worker"] # 该过滤规则只应用于 test 库中的 worker 表
+ignore-event = ["insert"] # 过滤掉 insert 事件
+ignore-sql = ["^drop", "add column"] # 过滤掉以 "drop" 开头或者包含 "add column" 的 DDL
+ignore-delete-value-expr = "name = 'john'" # 过滤掉包含 name = 'john' 条件的 delete DML
+ignore-insert-value-expr = "id >= 100" # 过滤掉包含 id >= 100 条件的 insert DML 
+ignore-update-old-value-expr = "age < 18" # 过滤掉旧值 age < 18 的 update DML
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:684]:
```suggestion
ignore-update-old-value-expr = "age < 18 or name = 'lili'" # 过滤掉旧值 age < 18 的 update DML
```

**Review by leoppro — APPROVED** [review]
LGTM

**Pull request merged**